system-addon/lib/SectionsManager.jsm

```diff
@@ -282,6 +280,13 @@ class SectionsFeed {
   onAddSection(event, id, options) {
     if (options) {
       this.store.dispatch(ac.BroadcastToContent({type: at.SECTION_REGISTER, data: Object.assign({id}, options)}));
+
+      // Make sure the section is in sectionOrder pref. Otherwise, prepend it.
+      const orderedSections = this.orderedSectionIds;
+      if (!orderedSections.includes(id)) {
+        orderedSections.splice(0, 0, id);
```

Comment:
nit: `orderedSections.unshift(id)` ;)

Reply:
*mindblown* I wonder who names these things

Reply:
Yeah! We should have `push` and `unpush` or `pop` and `unpop` for some consistency! ;) I would think `shift` is from https://www.unix.com/man-page/linux/1/shift/ and `unshift` is the pattern following `unget` http://heirloom.sourceforge.net/devtools/unget.1.html